teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super(name)
```

Comment:
- [ ] The other parameters needed to create a person need to be added since the Teacher class inherits from Person.

- [ ] The order of the arguments in the `super` method should be followed

  ```
  def initialize(specialization, age:, name: 'Unknown')
    super(age: age, name: name)
  ```

Reply:
Thanks @TedLivist